src/primitives.ts

```diff
@@ -78,6 +78,7 @@ export enum CustomUserProperties {
   ALL_WALLET_ADDRESSES_CONNECTED = 'all_wallet_addresses_connected',
   ALL_WALLET_CHAIN_IDS = 'all_wallet_chain_ids',
   BROWSER = 'browser',
+  CHAIN_ID = 'chain_id',
```

Comment:
Maybe to be more specific, `CURRENT_CHAIN_ID`?

Reply:
not a big deal, but this matches WALLET_ADDRESS and ALL_WALLET_ADDRESSES_CONNECTED so maybe it's fine to leave like this?